rhel/SPECS/pmm-managed.spec

```diff
@@ -54,7 +54,8 @@ export PMM_RELEASE_FULLCOMMIT=%{commit}
 export PMM_RELEASE_BRANCH=""
```

Comment:
Or we can set it in line 52 instead of %{full_pmm_version}

Reply:
I was only 99% sure that a new shell process is used for this block, not 100% :)